inference-engine/src/vpu/graph_transformer/src/backend/serialize.cpp

```diff
@@ -150,7 +150,10 @@ void BackEnd::serializeConstShapes(const Model& model, const mv_blob_header& blo
             serializeToBlob(data->desc().dims(), shapeLocation.dimsOffset);
         } else if (data->usage() == DataUsage::Output || data->usage() == DataUsage::Input) {
             auto ioDimsUpperBoundOffset = data->attrs().get<int>("ioDimsUpperBoundOffset");
-            serializeToBlob(data->desc().dims(), ioDimsUpperBoundOffset);
+            DimValues d = data->desc().dims();
+            if (d.has(Dim::N))
+                d.set(Dim::N, d.get(Dim::N, 1) * data->attrs().getOrDefault<int>("batch", 1));
+            serializeToBlob(d, ioDimsUpperBoundOffset);
```

Comment:
Could you please explain why it is needed? Since we change the way data is serialized, do we need de-serialization logic?

Reply:
Without this when you export blob with batch it will be imported back without batch. That is a bug

Reply:
Should we have a test on that?